src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static add(creature) {
+    this.alive.push(creature);
+  }
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.add(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden = false) {
+    super(name, health);
+
```

Comment:
you don't need constructor here, its already inherited

Reply:
I think in my case it's not an option, since I don't specify hidden property in Animal class.

Reply:
I guess you could add this property without constructor here